<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Resize</class>
 <widget class="QWidget" name="Resize">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QRadioButton" name="SizeNothingRadioButton">
     <property name="text">
      <string>Change nothing</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>144</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QRadioButton" name="SizePercentRadioButton">
     <property name="toolTip">
      <string>Check this to resize images by percentage</string>
     </property>
     <property name="text">
      <string>Size in %</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSlider" name="SizeXPercentHorizontalSlider">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QSpinBox" name="SizeXPercentSpinBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>width</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSlider" name="SizeYPercentHorizontalSlider">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QSpinBox" name="SizeYPercentSpinBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>height</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QRadioButton" name="SizePixelRadioButton">
     <property name="toolTip">
      <string>Check this to resize images by given size in pixel</string>
     </property>
     <property name="text">
      <string>Size in Pixel</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QSpinBox" name="SizeXPixelSpinBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>width</string>
     </property>
     <property name="maximum">
      <number>5000</number>
     </property>
     <property name="value">
      <number>1920</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>144</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QSpinBox" name="SizeYPixelSpinBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>height</string>
     </property>
     <property name="maximum">
      <number>5000</number>
     </property>
     <property name="value">
      <number>1080</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QComboBox" name="SizeRatioComboBox">
     <item>
      <property name="text">
       <string notr="true">1:1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">5:4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">4:3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">16:9</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QComboBox" name="PixelComboBox">
     <item>
      <property name="text">
       <string notr="true">VGA (640x480)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">XGA (1024x768)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">NTSC (576x486)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">PAL (720x486)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">HD (1920x1080)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">Flickr large (768x1024)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">iPhone landscape (480x320)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">SVGA (800x600)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">HD 720 (1280x720)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">2K (2048x1080)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">PSP (480x272)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">160x160</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>SizeXPercentHorizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SizeXPercentSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>238</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>343</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SizeYPercentHorizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SizeYPercentSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>238</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>343</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SizeXPercentHorizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SizeYPercentHorizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>238</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SizeXPercentSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SizeXPercentHorizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>343</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SizeYPercentSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SizeYPercentHorizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>343</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
